src/extras/PMREMGenerator.js

```diff
@@ -230,9 +262,9 @@ PMREMGenerator.prototype = {
 			magFilter: NearestFilter,
 			minFilter: NearestFilter,
 			generateMipmaps: false,
-			type: equirectangular ? equirectangular.type : UnsignedByteType,
-			format: equirectangular ? equirectangular.format : RGBEFormat,
-			encoding: equirectangular ? equirectangular.encoding : RGBEEncoding,
+			type: ( this._type !== undefined ) ? this._type : ( equirectangular ) ? equirectangular.type : UnsignedByteType,
+			format: ( this._format !== undefined ) ? this._format : ( equirectangular ) ? equirectangular.format : RGBEFormat,
+			encoding: ( this._encoding !== undefined ) ? this._encoding : ( equirectangular ) ? equirectangular.encoding : RGBEEncoding,
```

Comment:
...and remove this.

That is what I think, anyway...

Reply:
These sound reasonable, was trying to think if there was a use-case not covered. 
But I don't think so, and we can always adapt if necessary.

Reply:
Perhaps my comment is related to this. @WestLangley what do you think about the `sRGBEncoding` case?